### ISSUE TITLE
Camera doesn't work on Meizu 16x

### ISSUE THREAD

**Issue by trikrapka** [phhusson/treble_experimentations#871]
Hi, i have a problem on my Meizu 16 [Snapdragon 710]. I can't launch camera. Here is log.txt)
[camera.txt](https://github.com/phhusson/treble_experimentations/files/3825460/camera.txt)

**Comment by phhusson** [comment]
Why is your kernel running permissive SELinux?
v204 will have fixes for some errors found in your logs, so please test again with v204 when it gets released (should be within 2 days)

**Comment by trikrapka** [comment]
> Why is your kernel running permissive SELinux?
> v204 will have fixes for some errors found in your logs, so please test again with v204 when it gets released (should be within 2 days)

Is permissive SELinux bad? It always was running permissive selinux

**Comment by trikrapka** [comment]
> Why is your kernel running permissive SELinux?
> v204 will have fixes for some errors found in your logs, so please test again with v204 when it gets released (should be within 2 days)

No fixes( Camera, sound, calls, flashlight doesn't working as in v203. I will add new logs here
-Thanks for all u are doing for us

**Comment by phhusson** [comment]
> Is permissive SELinux bad? It always was running permissive selinux

Yes that's very bad security wise, but that's not really the question.
I'm worried that if you have permissive SELinux, your system has other weird stuff.
Why do you have selinux permissive?

**Comment by trikrapka** [comment]
> > Is permissive SELinux bad? It always was running permissive selinux
> 
> Yes that's very bad security wise, but that's not really the question.
> I'm worried that if you have permissive SELinux, your system has other weird stuff.
> Why do you have selinux permissive?

I don't know, I have Permissive on stock firmware. I haven't been changing it 😣

**Comment by phhusson** [comment]
This device isn't google certified?

**Comment by phhusson** [comment]
I mean, was it shipped with google apps?

**Comment by trikrapka** [comment]
> This device isn't google certified?

Yes. It isn't. It wasn't shipped with gapps. It was already in my stock firmware, (china flyme without gapps)

**Comment by phhusson** [comment]
Mmmm okay. I don't usually support non-Google certified devices, but i'll see what I can do.

Does everything work properly in my pie gsi, v121?

**Comment by trikrapka** [comment]
> Mmmm okay. I don't usually support non-Google certified devices, but i'll see what I can do.
> 
> Does everything work properly in my pie gsi, v121?

Yes. My device is certified on global rom, but I've installed china rom, where it isn't certified

**Comment by velv** [comment]
Well, I have a global version (certified) device with global firmware from meizu, but the list of bugs is about the same. I guess It does not matter

> Mmmm okay. I don't usually support non-Google certified devices, but i'll see what I can do.
> 
> Does everything work properly in my pie gsi, v121?

**Comment by phhusson** [comment]
well please provide logs on global firmware.
The fact that CN firmware is permissive shows it really isn't the same thing.

**Comment by phhusson** [comment]
Also I need logs right after boot

**Comment by phhusson** [comment]
Also v204 has several fixes for vendor 8.1, so you need to test it on  v204

**Comment by trikrapka** [comment]
> Also I need logs right after boot

here are, on Global firmware.
[txt.txt](https://github.com/phhusson/treble_experimentations/files/3882425/txt.txt)

**Comment by trikrapka** [comment]
Here are some logs on G
[sound.txt](https://github.com/phhusson/treble_experimentations/files/3882432/sound.txt)
[call.txt](https://github.com/phhusson/treble_experimentations/files/3882433/call.txt)
[camera.txt](https://github.com/phhusson/treble_experimentations/files/3882434/camera.txt)

**Comment by schocoladenpudding** [comment]
So, what is the result?

**Comment by trikrapka** [comment]
Nothing. Use mokee rom

чт, 9 апр. 2020 г., 15:55 schocoladenpudding <notifications@github.com>:
